components/button/src/vwc-button.ts

```diff
@@ -88,6 +91,15 @@ export class VWCButton extends MWCButton {
 		}
 	}
 
+	protected override update(changes:PropertyValues):void {
+		super.update(changes);
+		[...changes.keys()]
+			.filter(attributeName => ['name', 'value'].includes(attributeName as string))
+			.forEach((attributeName) => {
+				this.#_hiddenButton.setAttribute(attributeName as string, (this as unknown as GenericMap)[attributeName as string]);
+			});
+	}
+
 	protected updated(changes: Map<string, boolean>): void {
 		if (changes.has('type')) {
 			this.#_hiddenButton?.setAttribute('type', this.getAttribute('type') ?? '');
```

Comment:
I know it's not part of the PR, but if we refactor, we refactor :)
Why should `type` be handled here? Why can't it be in the `update` section?
Or why won't the others be in the `updated`? Just add `type` to the attributes array suggested above and delete the `updated`

Reply:
if it's all the same 2 u, let's do it on a different PR, as I'm not entirely familiar with the reasons that led to the current implementation 🙏

Reply:
for me personally, my code is a bit more readable.. @yinonov - can you help us decide which way to go?